setup.py

```diff
@@ -26,7 +26,6 @@
     package_dir={"": "python"},
     packages=setuptools.find_namespace_packages(where="python"),
     package_data={"": ["*.rst", "*.yaml"]},
-    data_files=[(data_files_path / "schema", ["schema/Watcher.yaml"])],
```

Comment:
Apparently one can only add comments at or near modified lines. It looks like tools_path, base_prefix and data_files_path are not needed anymore. If that's correct, please remove them and the pathlib import as well.